lib/dragonfly_audio/plugin.rb

```diff
@@ -1,56 +1,31 @@
 require 'dragonfly_audio/analysers/audio_properties'
+require 'dragonfly_audio/processors/tag'
 
 module DragonflyAudio
   class Plugin
-
     def call app, opts={}
       app.add_analyser :audio_properties, DragonflyAudio::Analysers::AudioProperties.new
 
```

Comment:
@asgerb wouldnt it make sense to use the AudioProprties class constants here so we dont list the same props in two places?